src/Util/Log/JUnit.php

```diff
@@ -51,7 +51,7 @@ final class JUnit extends Printer implements TestListener
     /**
      * @var bool
      */
-    private $reportRiskyTests = false;
+    private $reportRiskyTests;
```

Comment:
This value is immediately set in the constructor, but I think it's OK to keep the default property value here for clarity.